install-deps.sh

```diff
@@ -33,12 +33,45 @@ function install_seastar_deps {
 }
 
 function munge_ceph_spec_in {
-    # http://rpm.org/user_doc/conditional_builds.html
+    local for_what=$1
+    shift
     local OUTFILE=$1
-    sed -e 's/@//g' -e 's/%bcond_with make_check/%bcond_without make_check/g' < ceph.spec.in > $OUTFILE
+    sed -e 's/@//g' < ceph.spec.in > $OUTFILE
+    # http://rpm.org/user_doc/conditional_builds.html
     if [ $WITH_SEASTAR ]; then
         sed -i -e 's/%bcond_with seastar/%bcond_without seastar/g' $OUTFILE
     fi
+    case "$for_what" in
+	for_qa_run)
+	    sed -i -e 's/%bcond_with qa_run/%bcond_without qa_run/g' $OUTFILE
+	    ;&
```

Comment:
Using ‘;&’ in place of ‘;;’ causes execution to continue with the command-list associated with the next clause, if any. 

OK...